.gitignore

```diff
@@ -59,3 +59,15 @@ scripts/keycloak/data/*
 # ignore clowder chart deps
 charts/clowder2/charts
 charts/clowder2/*clowder2-software-dev.yaml
+
+# Environments
+.env
+.venv
+env/
+venv/
+ENV/
+env.bak/
+venv.bak/
+
+# Tests
+backend/app/tests/
```

Comment:
any reason we are not including tests to version control?

Reply:
Oh! I meant to ignore the additional outputs generated by tests. I will update this. Thanks.

Reply:
Updated. Please see. Thanks.

Reply:
Looks good. Approved :-)